### PR TITLE
 feat(presentational): add Menu component

### PR DESCRIPTION
**Pull request by agory**
Creation of menu component to render menu and recursively submenu.

### CODE REVIEW

**Comment by agory** [comment]
Pause as long as ft-list is not accepted

**Comment by agory** [comment]
need #36

File: src/web/hoc/Color/index.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ import PropTypes from 'prop-types';
 
 export default function colorify(Component) {
   function Color(props) {
-    const classes = props.className + props.color && ` is-${props.color}`;
-    return <Component {...props} className={classes} />;
+    const classes = props.className + (props.color && ` is-${props.color}`);
+    return <Component {...props} className={classes.trim()} />;
```
Comment (mfrachet) [inline src/web/hoc/Color/index.js]: `trim()`here is a trick ?

File: src/web/components/Menu/index.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Text from '../Text';
+import LinkText from '../LinkText';
+import List from '../List';
+import Item from '../Item';
+
+const renderItem = (item) => {
```
Comment (mfrachet) [inline src/web/components/Menu/index.js]: You can destructure on the `(item)` part 😊

File: src/web/hoc/Color/index.js (hunk 1 above)
Comment (agory) [reply src/web/hoc/Color/index.js]: The goal is to eliminate the gap between className and first class when className is empty.
But well if it's considered a trick :sob:  I can find another way :smile:

File: src/web/components/Menu/index.js (hunk 2 above)
Comment (agory) [reply src/web/components/Menu/index.js]: Oki i will do that :+1:

**Pull request merged**